context/src/repo/mod.rs

```diff
@@ -136,6 +148,37 @@ impl BundleRepo {
     }
 }
 
+#[cfg(feature = "pyo3")]
+#[pymethods]
+impl BundleRepo {
+    #[new]
+    fn py_new(
```

Comment:
Python explicitly needs a `new` constructor otherwise this cannot be instantiated in Python code. This struct needs to be create in Python since it can read from bundle uploads and check for validation.